### ISSUE TITLE
Use blank labels for activity

### ISSUE THREAD

**Issue by shaubert** [onfido/onfido-android-sdk#209]
#### What was the expected behaviour?
Use blank labels for activity

#### What happened instead?
    <application android:supportsRtl="false" >
        <meta-data
            android:name="com.google.mlkit.vision.DEPENDENCIES"
            android:value="face,ocr,barcode" />

        <activity
            android:name="com.onfido.android.sdk.capture.ui.OnfidoActivity"
            android:exported="false"
            android:hardwareAccelerated="true"
            **android:label="ᜀ"**
            android:launchMode="singleTask"
            android:screenOrientation="sensorPortrait"
            android:theme="@style/OnfidoActivityTheme" />
        <activity
            android:name="com.onfido.android.sdk.capture.ui.camera.CaptureActivity"
            android:exported="false"
            android:hardwareAccelerated="true"
            **android:label="ᜁ"**
            android:screenOrientation="sensorPortrait"
            android:theme="@style/OnfidoCaptureActivityStyle" />
        <activity
            android:name="com.onfido.android.sdk.capture.ui.camera.capture.ui.LivenessFlowStartActivity"
            android:exported="false"
            android:hardwareAccelerated="true"
            **android:label="ᜁ"**
            android:screenOrientation="sensorPortrait"
            android:theme="@style/OnfidoCaptureActivityStyle" />
    </application>`

#### Version info:
- `onfido-capture-sdk` version: 12.2.2

#### Other useful info:
<img width="407" alt="Screenshot 2022-08-01 at 11 22 20" src="https://user-images.githubusercontent.com/1733477/182669931-aaf7d29d-c8cc-43f6-b278-f6dbea66e7ff.png">

**Comment by Joren1204** [comment]
I'm having the same issue.
Before the capture screen is opened, the blank _OnfidoActivity_ is shown with the weird symbol in the toolbar.
Could the symbol be removed? Would look less like a bug...

```
OnfidoConfig.Builder(activity) //
	.withCustomFlow( //
		arrayOf( //
			DocumentCaptureStepBuilder.forNationalIdentity().withCountry(CountryCode.BE).build(),
			FaceCaptureStepBuilder.forPhoto().withIntro(true).build())) //
	.withAllowedDocumentTypes(listOf(DocumentType.NATIONAL_IDENTITY_CARD))
	.withSDKToken(token) //
	.build()```

**Comment by roy-ml** [comment]
Same here,
The symbol make it look like a bug.

https://user-images.githubusercontent.com/101554839/200243674-c4434049-495c-4399-a59c-c475be274ad7.mp4

**Comment by kushal-onfido** [comment]
Hello @shaubert @Joren1204 @roy-ml 
This issue has been fixed in the SDK release 15.0.0. Please refer to the release notes [here](https://github.com/onfido/onfido-android-sdk/releases/tag/15.0.0).

Thank you!
